### PR TITLE
Colormap pulldown for 2D histogram

### PR DESCRIPTION
**Pull request by marabuuu**
This PR solves Issue #225.
It creates a colormap dropdown menu in the advanced options section of the Plotter Widget (only visible when HISTOGRAM_2D is selected).

There, the user can choose between the following matplotlib colormaps:
- magma (default)
- viridis
- plasma
- PRGn
- PuOr
- coolwarm

Thank you so much @zoccoler and @haesleinhuepf for your help and guidance through the code!

### CODE REVIEW

**Comment by haesleinhuepf** [comment]
Awesome @marabuuu ! One idea: Do you think it is possible to replace our custom list of colormaps with the napari one. It took me a moment [to find it](https://github.com/napari/napari/blob/11c7d4854861950dc315da50b63b735cc3597ab4/napari/utils/colormaps/colormap_utils.py#L502), but I think `napari.utils.colormaps.ALL_COLORMAPS.keys()` could do the job. It's a list of colormap names supported by napari:

image

Big thanks for working on this!

**Comment by marabuuu** [comment]
Hi @haesleinhuepf,
Thanks for the hint. I will change it.

**Comment by marabuuu** [comment]
It should work now

**Comment by zoccoler** [comment]
Hi @marabuuu , 

I tested it and it is working, thanks!
I just added this line to auto-update the plot once the combobox changes, making its behavior similar to the other combobox.

I am just going to check if we can use the same container for both the plotting type and the colormap choice. Then, we could merge for next release!

File: napari_clusters_plotter/_Qt_code.py [hunk 1]
```diff
@@ -354,6 +354,41 @@ def algorithm_choice(name: str, value, options: dict, label: str):
     return container, choice_list
 
 
+def create_options_dropdown(name: str, value, options: dict, label: str):
```
Comment (zoccoler) [inline napari_clusters_plotter/_Qt_code.py:357]: I renamed this function to make it more general and replaced its references. I am keeping the `algorithm_choice` for backwards compatibility, although I doubt someone is actually using it directly.

**Review by zoccoler — APPROVED** [review]
I think this is working and it is ready to merge.

**Comment by haesleinhuepf** [comment]
Big thanks for working on this @marabuuu and @zoccoler !

**Pull request merged**